src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IReturnStatement.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Diagnostics;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Semantics;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests
+{
+    public partial class IOperationTests : SemanticModelTestBase
+    {
+        [Fact]
+        public void SimpleReturnFromRegularMethod()
+        {
+            string source = @"
+class C
+{
+    static void Method()
+    {
+        /*<bind>*/return;/*</bind>*/
+    }
+}
+";
+            string expectedOperationTree = @"IReturnStatement (OperationKind.ReturnStatement) (Syntax: 'return;')
+";
+            var expectedDiagnostics = DiagnosticDescription.None;
+            VerifyOperationTreeAndDiagnosticsForTest<ReturnStatementSyntax>(source, expectedOperationTree, expectedDiagnostics);
+        }
+
+        [Fact]
+        public void ReturnWithValueFromRegularMethod()
+        {
+            string source = @"
+class C
+{
+    static bool Method()
+    {
+        /*<bind>*/return true;/*</bind>*/
+    }
+}
+";
+            string expectedOperationTree = @"IReturnStatement (OperationKind.ReturnStatement) (Syntax: 'return true;')
```

Comment:
> @"IReturnStatement (OperationKind.ReturnStatement) (Syntax: 'return true;') [](start = 42, length = 76)

Please start the baseline from the beginning of the line, so that it would be clear what is the relationship between the nodes.